bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/MichellePrinter.java

```diff
@@ -134,42 +119,35 @@ private void writeImagingOutput(final ImagingResult result) {
 
         _println("");
 
-        // This object is used to format numerical strings.
-        final FormatStringWriter device = new FormatStringWriter();
-        device.setPrecision(2);  // Two decimal places
-        device.clear();
-
-        _print(CalculatablePrinter.getTextResult(result.sfCalc(), device));
-        _println(CalculatablePrinter.getTextResult(result.iqCalc(), device));
+        _print(CalculatablePrinter.getTextResult(result.sfCalc()));
+        _println(CalculatablePrinter.getTextResult(result.iqCalc()));
         _println("Sky subtraction aperture = " + result.observation().getSkyApertureDiameter() + " times the software aperture.\n");
 
         if (!instrument.polarimetryIsUsed()) {
-            _println(CalculatablePrinter.getTextResult(result.is2nCalc(), result.observation(), device));
+            _println(CalculatablePrinter.getTextResult(result.is2nCalc(), result.observation()));
         } else {
             _println("Polarimetry mode enabled.\n");
-            final String result2 = CalculatablePrinter.getTextResult(result.is2nCalc(), result.observation(), device);
+            final String result2 = CalculatablePrinter.getTextResult(result.is2nCalc(), result.observation());
             final String delims = "[ ]+";
             final String[] tokens = result2.split(delims);
             for (int i = 0; i < tokens.length; i++) {
```

Comment:
Agreed, this ain't pretty, but cleaning that up was not in the focus of this PR (replacing ```FormatStringWriter``` class).